src/ngraph/frontend/onnx_import/utils/convpool.hpp

```diff
@@ -92,25 +93,48 @@ namespace ngraph
              * @return nGraph node equivalent of the ONNX operation
              */
             template <class T>
-            inline NodeVector make_ng_pool(const Node& node)
+            inline NodeVector make_ng_pool(const Node& node, bool is_global = false)
```

Comment:
I'm not a fan of adding this `is_global` flag. Please refactor.
Let's have 2 public functions `make_ng_pool` and `make_ng_global_pool` and a third function `detail:: make_ng_pool` which has more implementation params.